test/src/main/java/org/axonframework/test/aggregate/StubAggregateLifecycle.java

```diff
@@ -102,6 +105,21 @@ public ApplyMore andThen(Runnable runnable) {
         };
     }
 
+    @Override
+    protected <D> ScheduleToken doScheduleDeadline(Instant triggerDateTime, D deadlineInfo) {
+        return null;
```

Comment:
Like you've already shared, I think it's a good idea to have a `StubDeadlineManager` in place.

Reply:
Yes, but not for this purpose. Until we really want to test scheduling with this `StubAggregateLifecycle`, `null` is fine.

Reply:
Actually, you are right. `StubDeadlineManager` should be used here.

Reply:
;p